docs/sphinx/developers/generating-test-images.txt

```diff
@@ -42,17 +42,18 @@ To generate a simple plate file:
 
 ::
 
-    touch "plate-test-file&plates=1&plateAcqs=1&plateRows=1&plateCols=1&fields=1.fake"
+    touch "simple-plate&plates=1&plateAcqs=1&plateRows=1&plateCols=1&fields=1.fake"
+    touch "default-plate&plates=1.fake"
+    touch "default-screen&screens=1.fake"
 
-This will create a single plate which is, by default, within a containing
-screen. To create a plate with no containing screen add ``&screens=0`` to the
-key-value pairs in the file name. As above a :file:`.fake.ini` file can be
-used.
+These will each create a single plate which is within a containing screen,
+by default in the first two cases. To create a plate with no containing screen
+add ``&screens=0`` to the key-value pairs in the file name. As above a
+:file:`.fake.ini` file can be used.
```

Comment:
Fixed below. I blame my extremely badly worded commit: https://github.com/ximenesuk/bioformats/commit/722701bd6d088f2537e1a10867583b33e705edce which states the opposite of what it does!